### DISCUSSION TITLE
Add support for hiding WhatsApp Status Updates

### DISCUSSION THREAD

**Discussion by githubbbbbbbbb** [d99kris/nchat#146] [Ideas]
Hello, please add an optinon to hide Status Updates in WA

**Comment by githubbbbbbbbb** [comment]
oh. sorry. fount the exact same discussion

**Comment by githubbbbbbbbb** [comment]
found the exact same discussion/ Closing as duplicate

**Comment by d99kris** [comment]
Hi @githubbbbbbbbb - actually, if you are referring to #137, I'm considering that request addressed.

Am I correct you'd like to be able to mute `Status Updates`, or hide it altogether? 
If either one of those, we can use this thread to track the request.

**Reply by githubbbbbbbbb** [reply]
Hi, probably even hide it. Just an option show_statuses 0/1 if that is possible

**Reply by d99kris** [reply]
Thanks, I'll look into it.

**Reply by gilcu3** [reply]
Is there any update on this? Just writing in case it has been forgotten in the backlog. Not being able to hide and mute that channel (`Status Updates`) makes the use of notifications a bit useless (I have too many contacts with status updates).

**Reply by d99kris** [reply]
Thanks for reminding, I'll try take a look soon.

**Reply by d99kris** [reply]
WhatsApp broadcast chat/messages are now **muted** by default as of 317652b. Edit `~/.nchatw/ui.conf` line `mute_status_broadcast` to change behavior.

**Reply by githubbbbbbbbb** [reply]
Great! An option to even hide status updates from chat list would be nice too

**Reply by acca** [reply]
I also support the possibility to eventually hide completely the `Status update` from the left sidebar. It remains at the top when there is any update and switching to other chats causes to read also the Status update, which BTW is moved at the top again... Can support in testing or developing it, I just need a pointer on where/how it could be implemented.

**Reply by d99kris** [reply]
Ok, looks like this is a popular request. Let me try address this soon. Wrt. providing pointers I prefer to implement it myself, as writing up steps/details and then reviewing PR iterations etc would maybe take more time. 😅 

**Edit:** To add on, as a temporary "workaround" one can configure muted chats to be sorted last, by setting:
```muted_position_by_timestamp=0```
and optionally disable the unread indicator for muted chats:
```muted_indicate_unread=0```

**Reply by d99kris** [reply]
Support for completely hiding status updates has been added in 35f1c95 - edit `ui.conf` and set [status_broadcast](https://github.com/d99kris/nchat#status_broadcast) to 0:
```
status_broadcast=0
```
